src/rest_scripts_POST.ecpp

```diff
@@ -66,6 +66,8 @@ UserInfo user;
         http_die("content-too-big", std::to_string(SCRIPT_FILE_MAX_SIZE).c_str());
     }
 
+    bool allowOverwrite = qparam.param("overwrite") == "true";
```

Comment:
I think I'd write this with parentheses to make operations priority explicit (there are too many languages juggled daily, all with their preferences, so sanity checker is blurred) and human-readable, but otherwise a neat gem :)